tasks/files-to-granules/schemas/config.json

```diff
@@ -0,0 +1,48 @@
+{
+  "title": "FilesToGranulesConfig",
+  "description": "Describes the config used by the files-to-granules task",
+  "type": "object",
+  "required": [
+    "input_granules"
+  ],
+  "properties": {
+    "input_granules": {
+      "type": "array",
+      "description": "granules object used to construct output for cumulus indexer",
+      "items": {
+        "type": "object",
+        "required": ["granuleId", "files"],
+        "properties": {
+          "granuleId": {
+            "type": "string"
+          },
+          "files": {
+            "type": "array",
+            "items": {
```

Comment:
From a quick glance, it looks like the only property of the incoming granules `files` that we care about is `filename`.  Do we need to specify the other fields here?

Reply:
No, c3ab57a